diff --git a/LazyScript/Localization.lua b/LazyScript/Localization.lua
index cdbbe92..9f16193 100644
--- a/LazyScript/Localization.lua
+++ b/LazyScript/Localization.lua
@@ -996,7 +996,7 @@ SEARCH_TARGET_UNDER = "Search for target under lvl "
 UNABLE_TO_DETERMINE = "unable to determine plus/minus sign"
 KEY = "Key: "
 VALUE = ", value: "
-KEY_2 = "Match found: Key: "
+--KEY_2 = "Match found: Key: "
 ONLY_CTRL_ALT_SHIFT = "Only Ctrl, Alt, and Shift are supported, not: "
 YOU_MUST_PUT = "You must put an action before -everyXXs."
 SYNTAX_EG = "syntax e.g. "
